dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -33,8 +33,7 @@
     StructuredOptions,
     UnstructuredOptions,
 )
-
-from . import utils as test_utils
+from dataprofiler.tests.profilers import utils as test_utils
```

Comment:
why did this need to change?

Reply:
the original line didn't import utils properly on my local, but the new line imports it consistently

Reply:
That might be bc the test was run from the dir as opposed to base of repo. I think here we are trying to separate from test vs in lib by using relative. but open to convo if we need a change here.